providers/aws/services/dynamodb/dynamodb_service_test.py

```diff
@@ -0,0 +1,178 @@
+                botocore_session=None,
+            ),
+            audited_account=AWS_ACCOUNT_NUMBER,
+            audited_user_id=None,
+            audited_partition="aws",
+            audited_identity_arn=None,
+            profile=None,
+            profile_region=None,
+            credentials=None,
+            assumed_role_info=None,
+            audited_regions=None,
+            organizations_metadata=None,
+        )
+        return audit_info
+
+    # Test Dynamo Service
+    @mock_dynamodb
+    def test_service(self):
+        # Dynamo client for this test class
+        audit_info = self.set_mocked_audit_info()
+        dynamodb = DynamoDB(audit_info)
+        assert dynamodb.service == "dynamodb"
+
+    # Test Dynamo Client
+    @mock_dynamodb
+    def test_client(self):
+        # Dynamo client for this test class
+        audit_info = self.set_mocked_audit_info()
+        dynamodb = DynamoDB(audit_info)
+        for client in dynamodb.regional_clients.values():
+            assert client.__class__.__name__ == "DynamoDB"
+
+    # Test Dynamo Session
+    @mock_dynamodb
+    def test__get_session__(self):
+        # Dynamo client for this test class
+        audit_info = self.set_mocked_audit_info()
+        dynamodb = DynamoDB(audit_info)
+        assert dynamodb.session.__class__.__name__ == "Session"
+
+    # Test Dynamo Session
+    @mock_dynamodb
+    def test_audited_account(self):
+        # Dynamo client for this test class
+        audit_info = self.set_mocked_audit_info()
+        dynamodb = DynamoDB(audit_info)
+        assert dynamodb.audited_account == AWS_ACCOUNT_NUMBER
+
+    # Test DynamoDB List Tables
+    @mock_dynamodb
+    def test__list_tables__(self):
+        # Generate DynamoDB Client
+        dynamodb_client = client("dynamodb", region_name=AWS_REGION)
+        # Create DynamoDB Tables
+        dynamodb_client.create_table(
+            TableName="test1",
+            AttributeDefinitions=[
+                {"AttributeName": "client", "AttributeType": "S"},
+                {"AttributeName": "app", "AttributeType": "S"},
+            ],
+            KeySchema=[
+                {"AttributeName": "client", "KeyType": "HASH"},
+                {"AttributeName": "app", "KeyType": "RANGE"},
+            ],
+            BillingMode="PAY_PER_REQUEST",
+        )
+        dynamodb_client.create_table(
+            TableName="test2",
+            AttributeDefinitions=[
+                {"AttributeName": "client", "AttributeType": "S"},
+                {"AttributeName": "app", "AttributeType": "S"},
+            ],
+            KeySchema=[
+                {"AttributeName": "client", "KeyType": "HASH"},
+                {"AttributeName": "app", "KeyType": "RANGE"},
+            ],
+            BillingMode="PAY_PER_REQUEST",
+        )
+        # DynamoDB client for this test class
+        audit_info = self.set_mocked_audit_info()
+        dynamo = DynamoDB(audit_info)
+        assert len(dynamo.tables) == 2
```

Comment:
Assert the possible values for the `tables` object.

Reply:
Done